clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1713,19 +1790,29 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
                 for (TopicPartition partition : partitions)
                     consumer.seek(partition, 0);
             }
+
+            @Override
+            public void onPartitionsLost(Collection<TopicPartition> partitions) {
+                System.out.println("hit partitions lost " + partitions);
```

Comment:
Is this an outdated debug statement? If not, maybe add a similar line to `onPartitionsRevoked()` ?